app/imports/startup/server/Mongo.js

```diff
@@ -1,9 +1,20 @@
 import { Meteor } from 'meteor/meteor';
 import { Stuffs } from '../../api/stuff/Stuff.js';
-
+import { dashboard } from '../../api/dashboard/dashboard';
 /* eslint-disable no-console */
 
 // Initialize the database with a default data document.
+/*
```

Comment:
I'd recommend not committing any commented code. If it's still commented and not used, it should be removed before merging to master.